pages/infrastructure_providers-network/resources.mdx

```diff
@@ -123,6 +123,30 @@ For **the deployment by DYDX token holders**:
 
   </Tab>
 </Tabs>
+
+## State Sync Service
+
+<Tabs items={['dydx-testnet-4', 'dydx-mainnet-1']}>
+  <Tab>
+For **Testnet**:
+
+| Team          | URI                                                                                     |
+| ------------- | --------------------------------------------------------------------------------------- |
+| Lavender.Five | https://services.lavenderfive.com/testnet/dydx/statesync |
+
+  </Tab>
+  <Tab>
+For **the deployment by DYDX token holders**:
+
+| Team          | URI                                                                                    |
+| ------------- | -------------------------------------------------------------------------------------- |
+| Polkachu | https://polkachu.com/state_sync/dydx |
+| Lavender.Five | https://services.lavenderfive.com/mainnet/dydx/statesync |
+| AutoStake | https://autostake.com/networks/dydx/#state-sync |
+
+  </Tab>
+</Tabs>
+
```

Comment:
In cases where we ask users to go to a providers and download something/ follow some brief instructions, I added those provider URLs to the Resources page. I think this is the right place for it until we reorganize the information architecture, but I do want to call it out in case someone with more context disagrees

Reply:
This seems fine to me